tests/parm/input_global_parent.nml.IN

```diff
@@ -0,0 +1,410 @@
+  kord_mt = 9
+  kord_wz = 9
+  kord_tr = 9
+  hydrostatic = .false.
+  phys_hydrostatic = .false.
+  use_hydro_pressure = .false.
+  beta = 0.
+  a_imp = 1.
+  p_fac = 0.1
+  k_split = 1
+  n_split = 5
+  nwat = @[NWAT]
+  na_init = @[NA_INIT]
+  d_ext = @[D_EXT]
+  dnats = @[DNATS]
+  fv_sg_adj = @[FV_SG_ADJ]
+  d2_bg = @[D2_BG]
+  nord = 2
+  nord_zs_filter = @[NORD_ZS_FILTER]
+  dddmp = @[DDDMP]
+  d4_bg = @[D4_BG]
+  vtdm4 = @[VTDM4]
+  delt_max = @[DELT_MAX]
+  ke_bg = 0.
+  do_vort_damp = .true.
+  external_ic = @[EXTERNAL_IC]
+  external_eta = .true.
+  gfs_phil = .false.
+  nggps_ic = @[NGGPS_IC]
+  mountain = @[MOUNTAIN]
+  ncep_ic = .false.
+  d_con = @[D_CON]
+  hord_mt = @[HORD_MT]
+  hord_vt = @[HORD_VT]
+  hord_tm = @[HORD_TM]
+  hord_dp = @[HORD_DP]
+  hord_tr = @[HORD_TR]
+  adjust_dry_mass = .false.
+  dry_mass = @[DRY_MASS]
+  consv_te = @[CONSV_TE]
+  do_sat_adj = @[DO_SAT_ADJ]
+  fast_tau_w_sec = @[FAST_TAU_W_SEC]
+  consv_am = .false.
+  fill = .true.
+  full_zs_filter = @[FULL_ZS_FILTER]
+  dwind_2d = .false.
+  print_freq = 6
+  warm_start = @[WARM_START]
+  no_dycore = .false.
+  z_tracer = .true.
+  agrid_vel_rst = .true.
+  read_increment = @[READ_INCREMENT]
+  res_latlon_dynamics = @[RES_LATLON_DYNAMICS]
+  do_schmidt = .true.
+  target_lat = @[TARGET_LAT]
+  target_lon = @[TARGET_LON]
+  stretch_fac = 1.0001
+/
+
+&external_ic_nml
+  filtered_terrain = .true.
+  levp = 128
+  gfs_dwinds = .true.
+  checker_tr = .false.
+  nt_checker = 0
+  
+/
+
+&surf_map_nml
+  cd2 = -1
+  cd4 = 0.15
+  max_slope = 0.15
+  n_del2_strong = 0
+  n_del2_weak = 15
+  n_del4 = 2
+  peak_fac = 1.
+  zero_ocean = .false.
+/
+
+&gfs_physics_nml
+  fhzero       = @[FHZERO]
+  h2o_phys     = @[H2O_PHYS]
+  ldiag3d      = @[LDIAG3D]
+  qdiag3d      = @[QDIAG3D]
+  lightning_threat = @[LIGHTNING_THREAT]
+  print_diff_pgr = @[PRINT_DIFF_PGR]
+  fhcyc        = @[FHCYC]
+  use_ufo      = .true.
+  pre_rad      = .false.
+  imp_physics  = @[IMP_PHYSICS]
+  iovr         = @[IOVR]
+  lcnorm       = @[LCNORM]
+  ltaerosol    = .true.
+  lradar       = .true.
+  ttendlim     = -999
+  dt_inner     = @[DT_INNER]
+  sedi_semi    = @[SEDI_SEMI]
+  decfl        = @[DECFL]
+  oz_phys      = @[OZ_PHYS_NEW]
+  oz_phys_2015 = @[OZ_PHYS_OLD]
+  lsoil_lsm    = @[LSOIL_LSM]
+  do_mynnedmf  = @[DO_MYNNEDMF]
+  do_mynnsfclay = @[DO_MYNNSFCLAY]
+  icloud_bl    = 1
+  bl_mynn_edmf = 1
+  bl_mynn_tkeadvect = @[BL_MYNN_TKEADVECT]
+  bl_mynn_edmf_mom = 1
+  do_ugwp      = @[DO_UGWP]
+  do_tofd      = @[DO_TOFD]
+  do_ca        = @[DO_CA]
+  gwd_opt      = @[GWD_OPT]
+  do_ugwp_v0   = @[DO_UGWP_V0]
+  do_ugwp_v1   = @[DO_UGWP_V1]
+  do_ugwp_v0_orog_only = @[DO_UGWP_V0_OROG_ONLY]
+  do_ugwp_v0_nst_only  = @[DO_UGWP_V0_NST_ONLY]
+  do_gsl_drag_ls_bl    = @[DO_GSL_DRAG_LS_BL]
+  do_gsl_drag_ss       = @[DO_GSL_DRAG_SS]
+  do_gsl_drag_tofd     = @[DO_GSL_DRAG_TOFD]
+  do_ugwp_v1_orog_only = @[DO_UGWP_V1_OROG_ONLY]
+  do_ugwp_v1_w_gsldrag = @[DO_UGWP_V1_W_GSLDRAG]
```

Comment:
Do we have a situation to set DO_UGWP_V1_W_GSLDRAG to .true.?

Reply:
In the regression tests, all gravity wave drag flags are configurable.

Reply:
ok, got it.